### ISSUE TITLE
About reproducing the proto net results

### ISSUE THREAD

**Issue by ChengJiacheng** [Sha-Lab/FEAT#4]
Hi~
The  reported result in the paper seems very competitive. But I have some troubles in reproducing them. 
Could you kindly share the settings (e.g. temperature) for achieving ProtoNet results reported in Table 1？

Also, I am a bit confused about the usage of temperature parameter.
In the paper, the temperature scalar is said to be used as
logits=  distances* temperature 
and temperature is chosen from {0.1, 1, 16, 32, 64, 128}.

But in the code, it seems to be used as 
logits =  distances/ temperature 
In this case, should the temperature still be chosen from {0.1, 1, 16, 32, 64, 128}?

Thanks!

**Comment by Han-Jia** [comment]
Hello,

The temperature used to scale the logits is in the form logits = distances/ temperature. In the experiments, the temperature is chosen from {0.1, 1, 16, 32, 64, 128}. 

In table 1, ProtoNet uses temperature 64.

**Comment by ChengJiacheng** [comment]
> Hello,
> 
> The temperature used to scale the logits is in the form logits = distances/ temperature. In the experiments, the temperature is chosen from {0.1, 1, 16, 32, 64, 128}.
> 
> In table 1, ProtoNet uses temperature 64.

Thanks a lot!
I will try it.

**Comment by ChengJiacheng** [comment]
> Hello,
> 
> The temperature used to scale the logits is in the form logits = distances/ temperature. In the experiments, the temperature is chosen from {0.1, 1, 16, 32, 64, 128}.
> 
> In table 1, ProtoNet uses temperature 64.

Hi, @Han-Jia,
Thanks for the information!

But the temperature 64 seems to not work for the 1-shot 5-way  ResNet case. 
I tried it and achieve 64.66%  best val acc and 59.65% test acc. 
Should I make any changes to other default parameters to reproduce the result in Table1 (61.40% test acc)?

any information will be greatly appreciated

**Comment by Han-Jia** [comment]
Hello, what are the concrete parameters you use?
Please try lr=0.001, T=64, step_size (schedulr) = 10, gamma=0.2; then you can get result around 61.4%.

**Comment by ChengJiacheng** [comment]
> Hello, what are the concrete parameters you use?
> Please try lr=0.001, T=64, step_size (schedulr) = 10, gamma=0.2; then you can get result around 61.4%.

Thanks again!
I can reproduce the result after setting lr as 0.001.
